lib/imgix/path.rb

```diff
@@ -108,5 +109,11 @@ def query
     def has_query?
       query.length > 0
     end
+
+    def validate_path!
+      if @path.match(/(\?.*)/)
```

Comment:
Is the `.*` even necessary? We could probably get away with just checking for the existence of a question mark.

Reply:
Good point 👍